src/js/Main/WatchLaterVideoWrapper.js

```diff
@@ -0,0 +1,158 @@
+import {
+  LOCAL_STORAGE_KEY,
+  CLASSNAME,
+  SNACKBAR_TEXT,
+} from "../constants/index.js";
+import { messenger, MESSAGE } from "../messenger/index.js";
+import { $, showSnackbar } from "../utils/index.js";
+import { WATCH_LATER_VIDEO_TEMPLATE, renderMainVideo } from "../Video/index.js";
+
+export default class WatchLaterVideoWrapper {
+  constructor() {
+    this.initializeVariables();
+    this.selectHTMLElements();
+    this.addMessageListeners();
+    this.addEventListeners();
+    this.render();
+  }
+
+  initializeVariables() {
+    this.watchLaterVideoItemsMap = new Map(
+      JSON.parse(
+        localStorage.getItem(LOCAL_STORAGE_KEY.WATCH_LATER_VIDEO_ITEMS)
+      )
+    );
+
+    this.watchLaterVideosMap = new Map();
+  }
+
+  selectHTMLElements() {
+    this.$noSavedVideoImage = $(`.${CLASSNAME.NO_WATCHING_VIDEO_IMAGE}`);
+    this.$watchLaterVideoWrapper = $(`.${CLASSNAME.WATCH_LATER_VIDEO_WRAPPER}`);
+  }
+
+  addMessageListeners() {
+    messenger.addMessageListener(MESSAGE.SAVE_VIDEO, this.saveVideo.bind(this));
+
+    messenger.addMessageListener(
+      MESSAGE.CANCEL_VIDEO_BUTTON_CLICKED,
+      this.deleteVideo.bind(this)
+    );
+
+    messenger.addMessageListener(
+      MESSAGE.MOVE_TO_WATCH_LATER_ICON_CLICKED,
+      this.saveVideo.bind(this)
+    );
+
+    messenger.addMessageListener(
+      MESSAGE.HIDE_IF_VIDEO_IS_SAVED,
+      this.hideIfVideoIsSaved.bind(this)
+    );
+  }
+
+  addEventListeners() {
+    this.$watchLaterVideoWrapper.addEventListener(
+      "click",
+      this.handleVideoWrapperClick.bind(this)
+    );
+  }
+
+  handleVideoWrapperClick(event) {
+    const { target: $videoWrapper } = event;
+    const { videoId } = $videoWrapper.parentElement.dataset;
+
+    if ($videoWrapper.classList.contains(CLASSNAME.MOVE_TO_HISTORY_ICON)) {
+      this.handleWatchedIconClick(videoId);
+      return;
+    }
+
+    if ($videoWrapper.classList.contains(CLASSNAME.DELETE_ICON)) {
+      this.handleDeleteIconClick(videoId);
+    }
+  }
+
+  handleWatchedIconClick(videoId) {
+    this.moveVideo({ videoId });
+    showSnackbar(SNACKBAR_TEXT.MOVED_TO_HISTORY_VIDEO);
+  }
+
+  handleDeleteIconClick(videoId) {
+    // eslint-disable-next-line no-alert
+    if (!window.confirm("정말 삭제하시겠습니까?")) {
+      return;
+    }
+    this.deleteVideo({ videoId });
+    messenger.deliverMessage(MESSAGE.SAVED_VIDEO_DELETED, { videoId });
+    showSnackbar(SNACKBAR_TEXT.VIDEO_DELETED);
```

Comment:
```suggestion
   if (!window.confirm("정말 삭제하시겠습니까?")) {
      return;
    }
    
    this.deleteVideo({ videoId });
    messenger.deliverMessage(MESSAGE.SAVED_VIDEO_DELETED, { videoId });
    showSnackbar(SNACKBAR_TEXT.VIDEO_DELETED);
```

개행문자를 넣어주는게 좋을 것 같아요!

Reply:
개행문자 추가하였습니다.